packages/framework/src/Model/Order/Item/OrderItemFacade.php

```diff
@@ -82,19 +84,28 @@ public function addProductToOrder($orderId, $productId)
     {
         $order = $this->orderRepository->getById($orderId);
         $product = $this->productRepository->getById($productId);
+        $orderDomainConfig = $this->domain->getDomainConfigById($order->getDomainId());
 
         $productPrice = $this->productPriceCalculationForUser->calculatePriceForUserAndDomainId(
             $product,
             $order->getDomainId(),
             $order->getCustomer()
         );
 
-        $orderProduct = $order->addProduct(
-            $product,
+        $orderProduct = $this->orderItemFactory->createProduct(
+            $order,
+            $product->getName($orderDomainConfig->getLocale()),
             $productPrice,
-            $this->orderItemFactory,
-            $this->domain,
-            $this->orderPriceCalculation
+            $product->getVat()->getPercent(),
+            static::DEFAULT_PRODUCT_QUANTITY,
```

Comment:
what about `PRODUCT::DEFAULT_PRODUCT_QUANTITY`  ? would be there any problem ? or $product->getDefaultProductQuantity and in product static::DEFAULT_PRODUCT_QUANTITY

Reply:
so you do not like the constant here? Why not? It is `OrderItemFacade` - it is responsible for manipulations with order items, it provides `addProductToOrder` method so it should be probably aware of the default quantity, or not? :thinking:

Reply:
i am not sure, but DEFAULT_PRODUCT_QUANTITY seems to me like domain of PRODUCT